### PR TITLE
Update data and types

### PR DESCRIPTION
**Pull request by cristian-aldea**
Currently, design changes and data additions are fragmented between different user stories. This leads to the user stories becoming bloated as they require changing many unrelated data structures and types.

This task aims to merge all of these changes quickly to make them uniform and available for all current user stories.

Minimal changes were done to the actual code to have the application in a working state

#49

### CODE REVIEW

**Review by RGPosadas — CHANGES_REQUESTED** [review]
- [x] Application still works as is, this is a nice Quality of Life refactoring/simplification

BLOCK: please add the issue number #49 on the first line of your commits, do not squash/fixup tho

**Review by RGPosadas — DISMISSED** [review]
LGTM

File: src/constants/poi.data.ts [hunk 1]
```diff
@@ -0,0 +1,1463 @@
...
+    image: require("../../assets/floors/MB-1.png")
+  }
+];
+
+export const POIInfo: POI[] = [
+  // ------------------
+  // LEVEL 8
+  // ------------------
+  {
+    id: "",
```
Comment (bit172) [inline src/constants/poi.data.ts]: The ids should be updated in the future

**Review by bit172 — DISMISSED** [review]
LGTM, POIs need some IDs,  but unblocking other tasks is a priority now

File: src/constants/poi.data.ts (hunk 1 above)
Comment (cristian-aldea) [reply src/constants/poi.data.ts]: Indeed, this pr mainly serves to unify the models and types, the actual values will be updated in the related user story.

Comment (TheGreatMarkus) [reply src/constants/poi.data.ts]: For consistency, I will set id's to be numbers. and still also fix a hidden error with the data I just noticed that

**Review by bit172 — DISMISSED** [review]

**Comment by cristian-aldea** [comment]
For consistency, I will set id's to be numbers. and still also fix a hidden error with the data I just noticed that didn't affect anything yet, but would have caused issues later

Sorry for reasking for reviews. You can just have a look at the changes from the latest commit, since its very simple.

File: src/constants/campus.data.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      latitude: 45.495376377001215,
+      longitude: -73.5777054168284,
+      latitudeDelta: 0.013275900094058102,
+      longitudeDelta: 0.009238533675670624
+    },
+    buildings: []
+  },
+  {
+    id: CampusId.Loyola,
+    displayName: "Loyola",
```
Comment (osahtout) [inline src/constants/campus.data.ts]: minor but maybe change to "Loyola Campus" to keep it consistent

**Review by osahtout — DISMISSED** [review]

File: src/constants/campus.data.ts (hunk 2 above)
Comment (cristian-aldea) [reply src/constants/campus.data.ts]: Good point, I'll do it now real quick, but again the actual values don't matter as much as the structure.

**Review by RGPosadas — APPROVED** [review]
LGTM

**Pull request merged**